mod-circulation/src/main/resources/vega/mod-circulation/features/requests.feature

```diff
@@ -45,3 +46,33 @@ Feature: Requests tests
     When method POST
     Then status 422
     And match $.errors[0].message == 'Page requests are not allowed for this patron and item combination'
+
+  Scenario: Given an item Id, a user Id, and a pickup location, attempt to create a hold request when the applicable request policy disallows holds
+    * def userBarcode = 'FAT-1031UBC'
+    * def itemBarcode = 'FAT-1031IBC'
+    * def extRequestTypesForRequestPolicy = ["Page", "Recall"]
+
+    # post a material type and policies
+    * call read('classpath:vega/mod-circulation/features/util/initData.feature@PostPolicies') { extRequestTypes: #(extRequestTypesForRequestPolicy) }
```

Comment:
there are 5 different policies in that method and it isn't really clear for which of them variable "extRequestTypes" are being sent. So I guess let's rename it into extRequestTypesFor...Policy

Reply:
fixed